scriptmodules/ports/openpht.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="openpht"
+rp_module_desc="OpenPHT is a community driven fork of Plex Home Theater"
+rp_module_section="opt"
+rp_module_flags="!arm"
+
+function install_bin_openpht() {
+local version="1.6.2.123-e23a7eef"
+local codename=$(lsb_release -cs)
+local package="openpht_${version}-${codename}_amd64.deb"
+local getdeb="https://github.com/RasPlex/OpenPHT/releases/download/v$version/$package"
+
+    if [[ "$__raspbian_name" == "wheezy"  ]]; then
+        md_ret_errors+=("The Debian package available is only for Jessie")
+        return 1
+    else
+        wget -nv -O "$__tmpdir/$package" $getdeb
+        if hasPackage "apt" "1.1" "ge"; then
+            apt install -y --allow-downgrades $__tmpdir/$package
+        else
+            # I feel dirty
```

Comment:
Although amusing, a better comment would be "falling back to dpkg" or similar no ? :)